README.md

```diff
@@ -1,3 +1,36 @@
-# mod1
-mod1
-this is added
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse Project
+
+> Training on github practice
+
+
+## Built With
+
+- Html & CSS
+
+## Authors
+
+👤 *Demes Ameneshoa*
+
+- GitHub: [@demesameneshoa](https://github.com/demesameneshoa)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+- Micronautes
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] As requested by the previous CR make user you use the[ MICROVERSE README TEMPLATE](https://github.com/microverseinc/readme-template) , then you should customize it with information about your project.
- [ ] Kindly avoid broken link in your README file 😄